requirements.yml

```
@@ -4,22 +4,20 @@
   version: 1.1.0
 
 - src: ome.ice
-  version: 3.0.0
+  version: 3.1.0
 
 - src: ome.java
   version: 2.0.3
 
-- src: ome.omego
-  version: 0.1.3
-
 - src: ome.omero_common
   version: 0.3.2
 
 - src: ome.omero_python_deps
   version: 2.0.1
 
-- src: ome.omero_server
-  version: 2.1.0
+- name: ome.omero_server
+  src: https://github.com/ome/ansible-role-omero-server/archive/8915ededb85b9d184849381d40ad388126e06283.tar.gz
```

```suggestion
  src: https://github.com/ome/ansible-role-omero-server/archive/605fe70f3a93e1fb4accfc485a5a6359d28181f0.tar.gz
```
Patch bin/omero to use venv3